test/broadcast.jl

```diff
@@ -419,7 +419,6 @@ struct Array19745{T,N} <: AbstractArray{T,N}
     data::Array{T,N}
 end
 Base.getindex(A::Array19745, i::Integer...) = A.data[i...]
-Base.setindex!(A::Array19745, v::Any, i::Integer...) = setindex!(A.data, v, i...)
```

Comment:
Can you split the example into two parts rather than changing it back? It's useful to test implementations that override `setindex!`.